src/udev/udev-builtin-net_id.c

```diff
@@ -286,6 +286,10 @@ static bool is_pci_multifunction(struct udev_device *dev) {
         return false;
 }
 
+static bool is_pci_ari_enabled(struct udev_device *dev) {
+        return udev_device_get_sysattr_value(dev, "ari_enabled");
```

Comment:
uh, this compiles without warning? you are implicitly converting a `const char*` to a `bool here`... Please use `!!` here to make the conversion explicitly.